### DISCUSSION TITLE
Pricing plans & users

### DISCUSSION THREAD

**Discussion by 3rgo** [statamic/cms#5483] [Q&A] [answered]
Hey everyone :wave: !

I'm currently researching options for a new project I'm thinking of, and Statamic may be the answer for a lot of things (great job on this project by the way).I only have one question that the website did not answer fully, and I believe some of you can be of help.

I want to build a kind of training courses app (think Laracasts), and I would like for my trainees to be able to create an account to follow their progress. So they willl not have access to the CP, and I don't anticipate anyone but me will use it, so I only need a single user account to manage it. 
I want to use the free Solo plan, since this is a hobby that probably won't earn me any money because the website will be fully free for everyone. 

Is there a way to fit this in the Solo plan ? Because the pricing page basically states that "Solo = 1 user account" and "Pro = Unlimited user accounts", but from what I could find, it does not specify if those user accounts are "basic users" (like I'm planning to use), or "editors/admins" (that would have access to CP).
I've been working with Laravel for a couple of years now, and I've read that Static can be integrated in a Laravel app, or extended with Laravel, so I guess this would be an option to consider if what I've understood about the plans.

So the question is : did I understand correctly the pricing plans ? And if so, is there a way to have "basic users" without paying the Pro plan ?

Thanks everyone !

TL;DR : Does the Solo plan allow for basic users (without access to the control panel) in addition to of the admin user ?

**Comment by jackmcdade** [comment, marked as answer]
You could absolutely create separate Laravel Users, that limitation is strictly referring to Statamic Users with control panel access.

**Reply by 3rgo** [reply]
> You could absolutely create separate Laravel Users, that limitation is strictly referring to Statamic Users with control panel access.
> […](#)
> On Mar 13, 2022, at 9:09 AM, Duncan McClean ***@***.***> wrote: ﻿ No, you can only have one user on the Solo plan. — Reply to this email directly, view it on GitHub, or unsubscribe. Triage notifications on the go with GitHub Mobile for iOS or Android. You are receiving this because you are subscribed to this thread.

Thanks for the reply Jack !
